### DISCUSSION TITLE
How to turn off 2fa verification?

### DISCUSSION THREAD

**Discussion by moondoor01** [community/community#134953] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello.
I am using 2fa verification in github. but it's really annoying.
I'd like to turn it off and gonna use password for login.
But I don't see how to do it.
Some say it's impossible to turn it off. 
Please answer my question.

**Comment by ebndev** [comment]
Hi @HideoYa!

Sorry to hear you are frustrated with this process. As of March 2023, we require all developers who contribute code on GitHub.com to enable two-factor authentication (2FA). You can read more information about this rollout in our blog post: [Raising the bar for software security: GitHub 2FA begins March 13](https://github.blog/news-insights/product-news/raising-the-bar-for-software-security-github-2fa-begins-march-13/).
